### ISSUE TITLE
Path Of Sorcery

### ISSUE THREAD

**Issue by Aslackez** [tiltedphoques/Mod-Compatibility#108]
# Mod information

#### Download link
https://www.nexusmods.com/skyrimspecialedition/mods/6660

#### Dependencies
- Unofficial Skyrim Special Edition Patch
- [https://www.nexusmods.com/skyrimspecialedition/mods/266](url)

# Compatibility
 >When you start casting restoration magic (healing, healing hands, heal other) everyone expect the person casting them gets CTD if  they are nearby.

# Additional Information
> Mod seems to work overall fine with perks rework and affect spell correctly.
> Still need further testing if some other perks or magic also cause crash.
> Script called IMP_RES__restoreHealth.pex seems to be cause for causing crash upon using restoration.

**Comment by DJLegends1011** [comment]
ight may need to retest this mod since there has been a couple of updates
also ever thought about testing sperg yet?

**Comment by hermyt** [comment]
Still crashes btw, tested on 1.41, healing spells ctd's any nearby clients aside from the caster

**Comment by DJLegends1011** [comment]
> Still crashes btw, tested on 1.41, healing spells ctd's any nearby clients aside from the caster

I've found the culprit since str and sensitive asf I've made a patch for it but mainly for vokriinator black to fix it

**Comment by hermyt** [comment]
> > Still crashes btw, tested on 1.41, healing spells ctd's any nearby clients aside from the caster
> 
> I've found the culprit since str and sensitive asf I've made a patch for it but mainly for vokriinator black to fix it

Got a link for the patch?  I was using vokriinator black too actually when I had to track down the crashing

**Comment by DJLegends1011** [comment]
> > > Still crashes btw, tested on 1.41, healing spells ctd's any nearby clients aside from the caster
> > 
> > 
> > I've found the culprit since str and sensitive asf I've made a patch for it but mainly for vokriinator black to fix it
> 
> Got a link for the patch? I was using vokriinator black too actually when I had to track down the crashing

currently got it stuck in my modpack since i'm making an excel page in which people can submit findings on what sync and what doesn't

**Comment by hermyt** [comment]
> > > > Still crashes btw, tested on 1.41, healing spells ctd's any nearby clients aside from the caster
> > > 
> > > 
> > > I've found the culprit since str and sensitive asf I've made a patch for it but mainly for vokriinator black to fix it
> > 
> > 
> > Got a link for the patch? I was using vokriinator black too actually when I had to track down the crashing
> 
> currently got it stuck in my modpack since i'm making an excel page in which people can submit findings on what sync and what doesn't

Which modpack?  is it a publicly available one?

**Comment by DJLegends1011** [comment]
> 

yes you can ask in str server for invite trying to personally fix vokrii black with others
